node/silkworm/common/secp256k1_context.hpp

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2020-2022 The Silkworm Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
```

Comment:
The C++ Core Guidelines recommend [usage of include guards over pragma once directive](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf8-use-include-guards-for-all-h-files) because the former are standard, the latter is not.
As a result, the rest of the project uses include guards all over the places.